### ISSUE TITLE
Blank lines are...  itty-bitty.  or wee.  or...

### ISSUE THREAD

**Issue by altendky** [supakeen/pinnwand#66]
https://bpaste.net/W2EQ
![image](https://user-images.githubusercontent.com/543719/77495147-1c3c2780-6e1e-11ea-9075-67510654a661.png)

**Comment by supakeen** [comment]
The blank lines are a clear bug however I see a second problem in your screenshot. Your line numbers aren't showing. Can you clear your cache/give it a hard reload/whatever we call it nowadays and if they are still gone let me know which browser and version you are using.

**Comment by altendky** [comment]
I just created another paste with the same issue (doesn't matter, but https://bpaste.net/RP3Q).  I reloaded it and the line numbers showed up and thus the blank lines were no longer itty-bitty.  I came here and reopened the above link and it has line numbers and non-wee blank lines.

Do we call this resolved?  Or is there more to check.  The browser is Chrome Version 80.0.3987.132 (Official Build) (64-bit) in Linux.
